src/index.ts

```diff
@@ -326,6 +330,28 @@ export async function install(
     };
     const packageBundle = await rollup.rollup(inputOptions);
     await packageBundle.write(outputOptions);
+
+    if (include && legacy) {
+      function renameImportPlugin() {
+        return {
+          name: 'rename-import-plugin',
+          resolveId(source) {
+            if (source.startsWith('/web_modules/')) {
+              return {
+                id: source.replace(/^\/web_modules\//, path.join(cwd, 'node_modules', path.sep)),
+              };
+            }
+            return null;
+          },
+        };
+      }
+      const noModuleBundle = await rollup.rollup({
+        input: glob.sync(include),
```

Comment:
I'm surprised that this worked, I would have assumed that you'd need to give a single "input" file as the app's entrypoint. Does `iife` just concatenate all the files together somehow?

Reply:
I’m surprised it worked too! You know I only tried it on an array of 1; I should test this on a bigger project

Reply:
Update: in a bigger project, `iife` can’t concatenate, because concatenation != bundling. See other comments; resolving this one for cleanup